jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/modal/ModalLiveDemo.java

```diff
@@ -0,0 +1,23 @@
+package io.github.com.sections.modal;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.bootstrap.elements.common.Button;
+import com.epam.jdi.light.ui.bootstrap.elements.common.Text;
+import com.epam.jdi.light.ui.bootstrap.elements.composite.Modal;
+
+public class ModalLiveDemo extends Section {
+
+    @UI("button[data-toggle='modal']")
+    public Button launchModal;
+    @UI(".modal-body")
+    public Text body;
+    @UI("//div[@class='modal-footer']//button[1]")
+    public Button closeButton;
+    @UI("//div[@class='modal-footer']//button[2]")
+    public Button saveButton;
+    @UI(".modal-header .close")
+    public Button closeX;
+    @UI("#exampleModalLive")
+    public Modal modalContent;
+}
```

Comment:
You should extend Modal class. It already have `title` field and `close()` method. Any child elements(body, closeButton, saveButton, modalContent) of your modal you should put in that class.